recipe/meta.yaml

```diff
@@ -32,7 +32,7 @@ requirements:
     - requests
     - scikit-image
     - toolz
-    - zarr >=2.8.1
+    - zarr <3
```

Comment:
We may want to add [a repodata patch]( https://github.com/conda-forge/conda-forge-repodata-patches-feedstock/blob/main/recipe/README.md ) for older version to ensure they can't be installed with Zarr v3

A simple example that could be tailored to our needs is PR: https://github.com/conda-forge/conda-forge-repodata-patches-feedstock/pull/937

Reply:
See https://github.com/conda-forge/conda-forge-repodata-patches-feedstock/pull/940

I'm unsure whether or not to include a napari-ome-zarr.yaml or just ping napari-ome-zarr to 0.10.3 (seems too restrictive)